geode-core/src/test/java/org/apache/geode/internal/cache/entries/AbstractRegionEntryTest.java

```diff
@@ -352,34 +356,83 @@ public void resolverShouldHandleConflictEventsFromDifferentDS() {
     verify(resolver, Mockito.times(1)).onEvent(any(), any());
   }
 
+  @Test
```

Comment:
Get rid of your changes to AbstractRegionEntryTest. It is a until test of AbstractRegionEntry so you do not need to be making it more complicated to test another class (in this case DiskStoreID). Just focus on testing the code you changed in DiskStoreID by unit testing it. We really don't want unit tests for other classes to be creating real instances of another class if it can be helped. So AbstractRegionEntryTest should not even show up as diffs on the pr. Just check it out and revert all your changes to it.